Resources/config/form_extension_legacy.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+	Author: Christian Raue <christian.raue@gmail.com>
+	Copyright: 2011-2015 Christian Raue
+	License: http://opensource.org/licenses/mit-license.php MIT License
+-->
+<container xmlns="http://symfony.com/schema/dic/services"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+	<services>
+		<service id="craue.form.flow.form_extension" class="Craue\FormFlowBundle\Form\Extension\FormFlowFormExtension">
+			<tag name="form.type_extension" alias="form" />
+		</service>
+
+		<service id="craue.form.flow.step_field_extension" class="Craue\FormFlowBundle\Form\Extension\FormFlowStepFieldExtension">
+			<tag name="form.type_extension" alias="hidden" />
```

Comment:
no need to create 2 files. You can specify ``extended-type`` and ``alias`` at the same time in the tag (Symfony 2.8 will just ignore ``alias`` if you also give the new extended type attribute)